docs/actions.md

```diff
@@ -2933,6 +2933,100 @@ Key | Description
 
 
 
+### automatic_code_signing
+
+Updates the Xcode 8 Automatic Codesigning Flag
+
+
+
+> Updates the Xcode 8 Automatic Codesigning Flag of all targets in the project
+
+automatic_code_signing | 
+-----|----
+Supported platforms | ios, mac
+Author | @mathiasAichinger, @hjanuschka
+Returns | The current status (boolean) of codesigning after modification
+
+
+
+<details>
+<summary>8 Examples</summary>
+
+```ruby
+# enable automatic code signing
+enable_automatic_code_signing
```

Comment:
Awesome 🎉  built-in new action from plugin!

The test failure is caused by method_missing because we don't find actions from aliases in `test_sample_code.rb` right now.
https://github.com/fastlane/docs/blob/984aa15ad655a638a0d07121cdf5a4b81aa3a73a/fastlane/actions/test_sample_code.rb#L43

So, we should also handle aliases in this PR?